### PR TITLE
Update Opera to 72.0.3815.207, use `curl` for downloading

### PR DESCRIPTION
**Pull request by supechicken666**
Update opera.rb frome 72.0.3815.148 to 72.0.3815.207
Fetch sha256 checksum automatically
Hide `tar` output

### CODE REVIEW

File: packages/opera.rb [hunk 1]
```diff
@@ -17,10 +17,10 @@ class Opera < Package
 
 
   def self.build
-    system "wget https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb"
-    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read("opera-stable_#{version}_amd64.deb") ) == '08ec3d1f1b142db4acaea9961687ad95db123f8567c07b0157e350aeed26a56d'
-    system "alien -tc opera-stable_#{version}_amd64.deb"
-    system "tar xvf opera-stable-#{version}.tgz"
+    system "curl https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb --output opera.deb"
```
Comment (uberhacker) [inline packages/opera.rb:20]: `wget` is easier. :)

File: packages/opera.rb [hunk 2]
```diff
@@ -17,10 +17,10 @@ class Opera < Package
...
+    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read("opera.deb") ) == `curl -L https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb.sha256sum`
```
Comment (uberhacker) [inline packages/opera.rb:21]: This is an interesting way to do it but it's probably easier just to insert the checksum verbatim.

File: packages/opera.rb [hunk 3]
```diff
@@ -17,10 +17,10 @@ class Opera < Package
...
+    system "alien -tc opera.deb > /dev/null"
+    system "tar xvf opera-stable-#{version}.tgz > /dev/null"
```
Comment (uberhacker) [inline packages/opera.rb:23]: I would rather not use `> /dev/null`.  This will hide any errors if they occur.

File: packages/opera.rb [hunk 4]
```diff
@@ -17,9 +17,10 @@ class Opera < Package
 
 
   def self.build
-    system "curl https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb --output opera.deb"
-    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read("opera.deb") ) == `curl -L https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb.sha256sum`
-    system "alien -tc opera.deb > /dev/null"
+    system "curl https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb -#o opera.deb"
```
Comment (uberhacker) [inline packages/opera.rb:20]: Change `-#o` to `-o`?

File: packages/opera.rb (hunk 3 above)
Comment (uberhacker) [reply packages/opera.rb:24]: Line 24 has `> /dev/null`.

File: packages/opera.rb (hunk 4 above)
Comment (supechicken) [reply packages/opera.rb:20]: `#` will make `curl` show the progress bar

File: packages/opera.rb (hunk 1 above)
Comment (supechicken) [reply packages/opera.rb]: `wget` is old now, and it can reduce one dependency as well

File: packages/opera.rb (hunk 2 above)
Comment (supechicken) [reply packages/opera.rb]: It makes updating the package easier, just correct the version number and done :)

File: packages/opera.rb (hunk 3 above)
Comment (supechicken) [reply packages/opera.rb:24]: Only `tar` output hide now, as `tar` won't occur error for the most time

Comment (supechicken) [reply packages/opera.rb:24]: `> /dev/null` won't hide error (it will hide `stdout` but keep `stderr`)

**Comment by supechicken** [comment]
Can we run `alien` as root to get rid of the warning message?
```shell
Rename all *.la files to *.la_tmp
############################################################################################################################### 100.0%
Archive downloaded
Warning: alien is not running as root!
Warning: Ownerships of files in the generated packages will probably be wrong.
opera-stable-72.0.3815.207.tgz generated
Preconfiguring package...
Rename all *.la_tmp files back to *.la
```

File: packages/opera.rb (hunk 1 above)
Comment (uberhacker) [reply packages/opera.rb]: We're using the latest wget in core.

File: packages/opera.rb (hunk 3 above)
Comment (uberhacker) [reply packages/opera.rb:24]: I get that but if there is an error, don't you need `stdout` to see where the error occurred?

File: packages/opera.rb (hunk 2 above)
Comment (uberhacker) [reply packages/opera.rb]: Fair enough. :)

**Comment by uberhacker** [comment]
No need to run as root since the chronos user already has write access to `/usr/local` which is the destination directory for all the files.

**Pull request merged**